InjectionBundle/InjectionClient.mm

```diff
@@ -103,9 +103,12 @@ + (void)load {
     // connect to InjetionIII.app using sicket
     if (InjectionClient *client = [self connectTo:INJECTION_ADDRESS])
         [client run];
-    else
+    else {
         printf("💉 Injection loaded but could not connect. Is InjectionIII.app running?\n");
-
+#ifndef __IPHONE_OS_VERSION_MIN_REQUIRED
```

Comment:
Will this condition take tvOS into consideration?

Reply:
Hmm no, does it need to. Can't tvOS make web requests by default?